### DISCUSSION TITLE
Qenv extension deprecated

### DISCUSSION THREAD

**Discussion by x140l31** [quasarframework/quasar#11462] [Quasar App Extensions] [answered]
Hi,

I see the message "Package no longer supported. Use 'build->env' in quasar.conf.js to do same thing." in Qenv Quasar App Extension. Why is that? 

.env files are often used to store sensitive credentials/tokens (for example Stripe key) and should be versioned.

**Comment by x140l31** [comment]
Ok there is already a thread [here](https://github.com/quasarframework/quasar/discussions/9318). Sorry, searcher didn't work for me

**Comment by hawkeye64** [comment, marked as answer]
@x140l31 It is also explained here in the Quasar documentation: https://quasar.dev/quasar-cli/handling-process-env#using-dotenv

Basically, the two app-extensions that were deprecated were useful, but with Quasar v2, the ability to handle your own (and all types of edge-cases that the app-extensions could not) made it so easy that the app-extensions were made obsolete.
